src/var/getDefaultOptions.js

```diff
@@ -45,7 +46,9 @@ function($, emojione, detectVersion, getSupportMode) {
             }
         };
 
-        if (getSupportMode(detectVersion(emojione)) > 4) {
+        var supportMode = !emojione ? getSupportMode(emojioneVersion) : getSupportMode(detectVersion(emojione));
+
+        if (supportMode > 4) {
```

Comment:
@mervick if emojione is not loaded, it will use emojioneVersion to get support mode, otherwise it will use emojione.
This ensures it works correctly if user has either included emojione lib manually, or set `window.emojioneVersion`.

Having included emojione lib would override `window.emojioneVersion` value which follows how the rest of the code works.

Reply:
As I see `getOptions()` used 2 times, first time it used in `loadEmojione()` function, and second in `init()`.
When it loads first, emojione not defined but filters option not used at all, in second time emojione will be defined as expected,
so that's must fix that issue.